src/lib/core/fiber.h

```diff
@@ -459,24 +460,14 @@ cord_slab_cache(void);
  * internal structures may be changed in a future, but
  * <box_region_*>() functions will remain API and ABI compatible.
  *
- * Data allocated on the region are guaranteed to be valid until
- * a call of a function from the certain set:
- *
- * - Related to transactions.
- * - Ones that may cause box initialization (box.cfg()).
- * - Ones that may involve SQL execution.
- *
- * FIXME: Provide more strict list of functions, which may
- * invalidate the data: ones that may lead to calling of
- * fiber_gc().
- *
- * It is safe to call simple box APIs around tuples, key_defs and
- * so on -- they don't invalidate the allocated data.
- *
  * Each fiber has its own box region. It means that a call of,
  * say, <box_region_used>() will give its own value in different
  * fibers. It also means that a yield does not invalidate data in
  * the box region.
+ *
+ * Before version 2.11 some API like executing a DML statement truncated
+ * fiber region to 0 so that missing box_region_truncate don't lead to leaks.
+ * Starting from 2.11 client code should truncates its allocations.
```

Comment:
Nit: Let's rephrase.

Prior to version 2.11, the box region was implicitly cleaned up on transaction commit (see <box_txn_commit>()) so that <box_region_truncate>() wasn't strictly necessary. Starting from version 2.11, it isn't true anymore, and the client code must free all its allocations explicitly.

Reply:
Ok.